src/core/Table/EditableCell.tsx

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Cell, TableCellProps } from 'react-table';
+
+export type OnCellEditCallbackParams<T extends Record<string, unknown>> = {
+  columnId: string;
+  value: string;
+  rowData: T;
+};
+
+export type EditableCellProps<T extends Record<string, unknown>> = {
+  cell: Cell<T>;
+  cellProps: TableCellProps;
+  onCellEdit?: (cellEditParams: OnCellEditCallbackParams<T>) => void;
```

Comment:
I am not sure whether to make params object or just each param separately `(columnId, value, rowData) => void`.
Currently made it as a object because I saw it somewhere like that 😀

Reply:
object is less clear to me

Reply:
Changed to just params